src/components/Navigation/styles.scss

```diff
@@ -0,0 +1,133 @@
+// Mobile portrait
+.wrapper {
+	width: 100%;
+	height: 100%;
+	font-size: 1.5rem;
+	background-color: red;
+}
+
+	.button {
+		display: flex;
+		position: absolute;
+		top: 0.85em;
+		right: 0.75em;
+		height: var(--button_height);
+		align-items: center;
+		font-size: 1em;
+		text-transform: uppercase;
+		--button_height: 1.15em;
+
+		&:before {
+			width: 1em;
+			height: var(--button_height);
+			content: "";
+			background-color: blue;
+		}
+	}
+
+		.label {
+			width: 3.8em;
+			margin-left: 0.75em;
+
+
+			&[aria-hidden="true"]{
+				opacity: 0;
+			}
+			&[aria-hidden="false"]{
+				opacity: 1;
+			}
+
+			&:nth-child(1){
+				position: absolute;
+				right: 0;
+			}
+		}
+
+	.menu {
+		display: flex;
+		padding-top: 20vh;
+		flex-direction: column;
+		align-items: center;
+		list-style: none;
+	}
+
+		.link {
+			font-size: 2.8em;
+
+			a {
+				text-decoration: none;
+			}
+		}
+	
+// Mobile landscape
+@media (orientation: landscape) {
+	.wrapper {
+		font-size: 1rem;
+		background-color: red;
+	}
+
+		.menu {
+			padding-top: 10vh;
+			line-height: 1.25;
+		}
+}
+
+// Tablet portrait
+@media (min-width: 768px) {
+	
+	.wrapper {
+		font-size: 1.75rem;
+	}
+
+		.button {
+			right: 50%;
+			transform: translateX(50%);
+		}
+
+		.menu {
+			padding-top: 15vh;
+		}
+
+			.link {
+				margin-bottom: 1rem;
+			}
+}
+
+// Tablet landscape
+@media (min-width: 768px) and (orientation: landscape) {
+	.wrapper {
+		font-size: 1.5rem;
+	}
+
+			.link {
+				margin-bottom: 0;
+			}
+}
+
+// Desktop portrait
+@media (min-width: 1200px) and (max-aspect-ratio: 8/9) {
+
+}
+
+// Desktop landscape
+@media (min-width: 1200px) {
+	.wrapper {
+		font-size: 1.2rem;
+	}
+
+		.button {
+			top: 1.875rem;
+		}
+
+		.menu {
+			padding-top: 20vh;
+		}
+
+			.link {
+				font-size: 4.5em;
```

Comment:
Once you've re-done all the scaling maths etc, whatever the result is should go into the `.container` (`.menu` currently) rather than on the individual item.  CSS is all about the cascade - so you want to specify stuff as it makes sense to - specifying on the `.item` here would only make sense if the parent already had a different `font-size` and you were scaling off of that, if you get me?